crates/nu-cmd-lang/src/core_commands/echo.rs

```diff
@@ -43,6 +43,16 @@ little reason to use this over just writing the values as-is."#
         _input: PipelineData,
     ) -> Result<PipelineData, ShellError> {
         let args = call.rest(engine_state, stack, 0);
+        report_error_new(
+                engine_state,
+            &ShellError::GenericError(
+                "Deprecated command".into(),
+                "`echo` is deprecated and will be removed in 0.85.".into(),
+                Some(call.head),
+                Some("Please use the `print` command to print the data to the terminal or directly the value if you want to use it in a pipeline.".into()),
```

Comment:
Wording:

"Please use the `print` command to print the data to the terminal or use the value directly to send it to the pipeline."

We should probably also add some examples to the deprecation text

```
To print to the screen:
  print "hello world"
To send to the pipeline:
  "hello world" | str length
```

Reply:
i can do that for sure :yum:

Reply:
i've fixed the wording in a9f9a9445 and with b31d29027, the deprecation message is now the following :innocent: 
```nushell
> echo "foo"
Error:   × Deprecated command
   ╭─[entry #1:1:1]
 1 │ echo "foo"
   · ──┬─
   ·   ╰── `echo` is deprecated and will be removed in 0.85.
   ╰────
  help: Please use the `print` command to print the data
        to the terminal or use the value directly to send
        it to the pipeline.

            To print to the screen:
                print "hello world"

            To send to the pipeline:
                "hello world" | str length


foo
```